src/fermilib/transforms/_bksf.py

```diff
@@ -0,0 +1,449 @@
+                                continue
+
+                    if (p == q) or (r == s):
+                        continue
+
+                    # Handle case of four unique indices.
+                    elif len(set([p, q, r, s])) == 4:
+
+                        if coefficient2 and p >= q:
+                            edge_matrix[p, q] = bool(complex(iop[p, q, r, s]))
+                            a, b = sorted([r, s])
+                            edge_matrix[b, a] = bool(complex(iop[p, q, r, s]))
+
+                    # Handle case of three unique indices.
+                    elif len(set([p, q, r, s])) == 3:
+
+                        # Identify equal tensor factors.
+                        if p == r:
+                            a, b = sorted([q, s])
+                            edge_matrix[b, a] = bool(complex(iop[p, q, r, s]))
+                        elif p == s:
+                            a, b = sorted([q, r])
+                            edge_matrix[b, a] = bool(complex(iop[p, q, r, s]))
+                        elif q == r:
+                            a, b = sorted([p, s])
+                            edge_matrix[b, a] = bool(complex(iop[p, q, r, s]))
+                        elif q == s:
+                            a, b = sorted([p, r])
+                            edge_matrix[b, a] = bool(complex(iop[p, q, r, s]))
+
+                    # Handle case of two unique indices.
+                    elif len(set([p, q, r, s])) == 2:
+                        if p == s:
+                            a, b = sorted([p, q])
+                            edge_matrix[b, a] = bool(complex(iop[p, q, r, s]))
+
+                        else:
+                            a, b = sorted([p, q])
+                            edge_matrix[b, a] = bool(complex(iop[p, q, r, s]))
+
+    return edge_matrix.transpose()
+
+
+def one_body(edge_matrix_indices, p, q):
+    """Map the term a^\dagger_p a_q + a^\dagger_q a_p to QubitOperator.
+    The definitions for various operators will be presented in a paper soon
+
+    Args:
+        edge_matrix_indices(numpy array): Specifying the edges
+        p and q (int): specifying the one body term.
+
+    Return: 
+        An instance of QubitOperator()
+    """
+    # Handle off-diagonal terms.
+    qubit_operator = QubitOperator()
+    if p != q:
+        a, b = sorted([p, q])
+        B_a = edge_operator_b(edge_matrix_indices, a)
+        B_b = edge_operator_b(edge_matrix_indices, b)
+        A_ab = edge_operator_aij(edge_matrix_indices, a, b)
+        qubit_operator += (-1j/2.) * (A_ab * B_b + B_a * A_ab)
+
+    # Handle diagonal terms.
+    else:
+        B_p = edge_operator_b(edge_matrix_indices, p)
+        qubit_operator += (QubitOperator((), 1) - B_p) / 2.
+
+    return qubit_operator
+
+
+def two_body(edge_matrix_indices, p, q, r, s):
+    """Map the term a^\dagger_p a^\dagger_q a_r a_s + h.c. to QubitOperator.
+    The definitions for various operators will be covered in a paper soon.
```

Comment:
I can add my information, and I will comeback and update the code when the paper is published.
Or, as you suggested waiting for the arxiv link is also fine with me.